.github/workflows/tiobe_scan.yml

```diff
@@ -0,0 +1,48 @@
+name: TICS run self-hosted test (github-action)
+
+on:
+  pull_request:
+  workflow_dispatch: # Allows manual triggering
+  schedule:
+    - cron: "0 2 * * 6" # Every Saturday 2:00 AM UTC
+
+jobs:
+  build:
+    runs-on: [self-hosted, linux, amd64, tiobe, jammy]
+
+    steps:
+      - name: Checkout the project
+        uses: actions/checkout@v4
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v5.3.0
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          pip install tox
+          pip install pylint flake8
+
+      - name: Run tox tests to create coverage.xml
+        run: |
+          tox run -e katib-controller-unit
+          tox run -e katib-db-manager-unit
```

Comment:
are we OK with not including katib-ui which has essentially 0% cover?

Reply:
we actually don't have even the unit environment. I have not included it because there is no unit env and unittets implemented. I don't think it is good but it is not for this ticket to address